cypress/pages/ndv.ts

```diff
@@ -10,6 +10,7 @@ export class NDV extends BasePage {
 		inputOption: () => cy.getByTestId('ndv-input-option'),
 		inputPanel: () => cy.getByTestId('ndv-input-panel'),
 		outputPanel: () => cy.getByTestId('output-panel'),
+		inputDataContainer: () => this.getters.inputPanel().getByTestId('ndv-data-container'),
```

Comment:
`inputDataContainer` and `outputDataContainer` are identical because you can't chain `cy.get`, which is what `getByTestId` essentially is. So only the last `getByTestId` will yield the value.  So this would evaluate as `true`:
``` js
		ndv.getters.inputDataContainer().then(inputData => {
			ndv.getters.outputDataContainer().then(outputData => {
				expect(inputData.text()).to.eq(outputData.text());
			})
		})
```
<img width="679" alt="CleanShot 2023-01-17 at 16 43 19@2x" src="https://user-images.githubusercontent.com/12657221/212943608-6deff019-6ca2-4a47-9886-028bc8756af0.png">

Instead you'd have to use find to look for child of that element. We have a helper command which does this — `findChildByTestId` so you could do:
```
this.getters.inputPanel().findChildByTestId('ndv-data-container')
```

Reply:
thanks. updated